docs/installation.rst

```diff
@@ -21,14 +21,22 @@ BookBrainz depends on having PostgreSQL, Redis, Elasticsearch and NodeJS set up
 
 But first some basic configuration common to both docker and manual installation.
 
+Forking
+*******
+
+Fork this repository
+
+``https://github.com/metabrainz/bookbrainz-site.git``
+
+
 Cloning
 *******
 
 To clone the repository and point the local HEAD to the latest commit in the stable branch, something like the following command should work
 
 ::
     
-    git clone --recurse-submodules https://github.com/metabrainz/bookbrainz-site.git
+    git clone --recurse-submodules https://github.com/<Your_Github_Username>/bookbrainz-site.git
```

Comment:
Actually, while we are here, I notice we still have submodules instructions while not using submodules anymore
```suggestion
    git clone https://github.com/<Your_Github_Username>/bookbrainz-site.git
```